prometheus-exporters/image-usage-exporter/templates/metrics-podmonitor.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+
+metadata:
+  name: {{ include "image-usage-exporter.fullname" . }}-exporter
```

Comment:
This will result in a duplicate `-exporter` at the end like `image-usage-exporter-exporter`